Sources/Commands/Options.swift

```diff
@@ -61,5 +61,8 @@ public class ToolOptions {
     /// Whether to enable code coverage.
     public var shouldEnableCodeCoverage = false
 
+    /// Whether to display code coverage.
+    public var showCodeCoverage = false
+
```

Comment:
We should get rid of showing code coverage for now, users can just look at the JSON file themselves.